configs/tsconfig.backend.json

```diff
@@ -2,6 +2,6 @@
   "extends": "./tsconfig.base.json",
   "compilerOptions": {
     "typeRoots": ["../", "node_modules/@types"],
-    "types": ["types/backend", "node"]
+    "types": ["types/backend"]
```

Comment:
why did you remove node types ?

Reply:
I added it back for now(as you said, it's not part of this task) but I spoke with the runtime time again and they said we don't want to let our user know the code is running in Node.js on propose and which Node.js modules the user can use. This is way we couldn't find this information in our articles.
Not sure if we need to deal with this question now, or we should just need to remove this support for now and add a task to our backlog(currently the online IDE doesn't support this syntax(`require`) and doesn't dive autocomplete for node.js module)